### ISSUE TITLE
Version release 2.0.0

### ISSUE THREAD

**Issue by kangz12345** [snu-quiqcl/qiwis#169]
I think we have to release a new version tag 2.0.0 as we updated the readme!
Please resolve this as you have done it before @BECATRUE !

**Comment by BECATRUE** [comment]
I saved the draft of this release!
Please check it in 'Releases' tab. @kangz12345

**Comment by kangz12345** [comment]
I checked it out and tried the autogeneration of _What's changed_.
If you have any opinion about this, please let me know!
@BECATRUE

**Comment by kangz12345** [comment]
I added the `bug` tag because I got the following error when I tried to install `qiwis` by the _recommended_ way in the README:
```
% pip install git_https://github.com/snu-quiqcl/qiwis.git@v2.0.0
ERROR: Invalid requirement: 'git_https://github.com/snu-quiqcl/qiwis.git@v2.0.0'
Hint: It looks like a path. File 'git_https://github.com/snu-quiqcl/qiwis.git@v2.0.0' does not exist.
```

At first I thought this is because we did not publish the version 2.0.0, but I found that there was a typo: `git_https`.
If I try it with `git+https`, I can see some thing more:
```
Collecting git+https://github.com/snu-quiqcl/qiwis.git@v2.0.0
  Cloning https://github.com/snu-quiqcl/qiwis.git (to revision v2.0.0) to /private/var/folders/w1/l25vrwxj6hg9ljrgry422bxc0000gn/T/pip-req-build-xrjcf8vb
  Running command git clone --filter=blob:none --quiet https://github.com/snu-quiqcl/qiwis.git /private/var/folders/w1/l25vrwxj6hg9ljrgry422bxc0000gn/T/pip-req-build-xrjcf8vb
  WARNING: Did not find branch or tag 'v2.0.0', assuming revision or ref.
  Running command git checkout -q v2.0.0
  error: pathspec 'v2.0.0' did not match any file(s) known to git
  error: subprocess-exited-with-error
  
  × git checkout -q v2.0.0 did not run successfully.
  │ exit code: 1
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× git checkout -q v2.0.0 did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
```

Therefore, I will update `git_https:` -> `git+https:` in the README.
Do you agree with this? @BECATRUE

**Comment by BECATRUE** [comment]
> I checked it out and tried the autogeneration of _What's changed_.
> If you have any opinion about this, please let me know!
> @BECATRUE 

Wow, I didn't know there was such a function. It's wonderful!

**Comment by BECATRUE** [comment]
> I added the `bug` tag because I got the following error when I tried to install `qiwis` by the _recommended_ way in the README:
> 
> ```
> % pip install git_https://github.com/snu-quiqcl/qiwis.git@v2.0.0
> ERROR: Invalid requirement: 'git_https://github.com/snu-quiqcl/qiwis.git@v2.0.0'
> Hint: It looks like a path. File 'git_https://github.com/snu-quiqcl/qiwis.git@v2.0.0' does not exist.
> ```
> 
> At first I thought this is because we did not publish the version 2.0.0, but I found that there was a typo: `git_https`. If I try it with `git+https`, I can see something more:
> 
> ```
> Collecting git+https://github.com/snu-quiqcl/qiwis.git@v2.0.0
>   Cloning https://github.com/snu-quiqcl/qiwis.git (to revision v2.0.0) to /private/var/folders/w1/l25vrwxj6hg9ljrgry422bxc0000gn/T/pip-req-build-xrjcf8vb
>   Running command git clone --filter=blob:none --quiet https://github.com/snu-quiqcl/qiwis.git /private/var/folders/w1/l25vrwxj6hg9ljrgry422bxc0000gn/T/pip-req-build-xrjcf8vb
>   WARNING: Did not find branch or tag 'v2.0.0', assuming revision or ref.
>   Running command git checkout -q v2.0.0
>   error: pathspec 'v2.0.0' did not match any file(s) known to git
>   error: subprocess-exited-with-error
>   
>   × git checkout -q v2.0.0 did not run successfully.
>   │ exit code: 1
>   ╰─> See above for output.
>   
>   note: This error originates from a subprocess, and is likely not a problem with pip.
> error: subprocess-exited-with-error
> 
> × git checkout -q v2.0.0 did not run successfully.
> │ exit code: 1
> ╰─> See above for output.
> 
> note: This error originates from a subprocess, and is likely not a problem with pip.
> ```
> 
> Therefore, I will update `git_https:` -> `git+https:` in the README. Do you agree with this? @BECATRUE

You are right! I think I made a mistake in the process of moving the url.😥
I also agree with `git+https:` and is there any more review?

**Comment by kangz12345** [comment]
I am okay with the release draft. Let's release it after I fix the README.

**Comment by BECATRUE** [comment]
Okay! Please fix it.

**Comment by BECATRUE** [comment]
I released `v2.0.0` and installed it using `pip install ...`, but I failed importing it or using the `qiwis` command.
Do you have the same symptom? @kangz12345

**Comment by kangz12345** [comment]
I have the same issue, but for the command, it fails because of the importing issue (so the command is registered correctly).
The problem is that the python cannot read the installed `qiwis` module.

**Comment by BECATRUE** [comment]
Oh, I refreshed the command shell and the problem was resolved.
I don't know what the problem is.😂

**Comment by BECATRUE** [comment]
Also, I saved the draft of `v2.0.1` including the updated README file.
Please check it! @kangz12345

**Comment by kangz12345** [comment]
Okay I confirm that the solution works for me as well!
